vagrant/ansible/roles/common.prep/tasks/centos9.yml

```diff
@@ -0,0 +1,12 @@
+---
+# required for python3-pyxattr
+- name: Enable CRB repo
+  command: dnf config-manager --set-enabled crb
+
+- name: Enable Samba nightly rpms repository
+  command: >-
+    yum-config-manager --add-repo
+    http://artifacts.ci.centos.org/samba/pkgs/master/centos/samba-nightly-master.repo
+
+- name: Restart systemd
+  command: kill -TERM 1
```

Comment:
This was added for a limitation in CentOS  8 and we had to add this workaround to get around a problem we hit then. This may no longer be a problem with Centos 8 or even CentOS9. Let's remove it for Centos9 and see if we hit the problem again. If yes, we know what works around the problem :)

Reply:
Thanks. I'll remove it for CentOS 9

Reply:
> This was added for a limitation in CentOS 8 and we had to add this workaround to get around a problem we hit then. This may no longer be a problem with Centos 8 or even CentOS9. Let's remove it for Centos9 and see if we hit the problem again. If yes, we know what works around the problem :)

I could successfully complete a sanity test run without the above systemd restart task. I've created #34 to get it removed.